keps/sig-network/1669-proxy-terminating-endpoints/README.md

```diff
@@ -206,6 +208,11 @@ and internalTrafficPolicy.
 * Metrics to publish how many Services/Endpoints are routing traffic to terminating endpoints.
 * Manual or automated rollback testing (see [Test Plan](#test-plan) section)
 
+#### GA
+
+* Feedback from users in issue [85643](https://github.com/kubernetes/kubernetes/issues/85643) that enabling ProxyTerminatingEndpoints can achieve zero downtime rolling updates when using `externalTrafficPolicy: Local`
+* E2E tests demonstrating that rolling updates can be performed with negligible downtime (ideally 100%, but may not feasible without flakiness) with a Service LoadBalancer when using externalTrafficPolicy: Local
+
```

Comment:
L254: too late now, but
(a) we should have linked them here before
(b) enablement/disablement is technically a test that switches the feature on/off in the middle of the test to validate what happens there - I think you were talking about feature tests themselves